rust/connlib/libs/tunnel/src/tun_android.rs

```diff
@@ -157,7 +156,7 @@ impl IfaceConfig {
     }
 
     pub async fn up(&mut self) -> Result<()> {
-        log::debug!("`up` unimplemented on Android");
+        tracing::debug!("`up` unimplemented on Android");
```

Comment:
Nit: The module path of this log statement will already include `tun_android` thus we could just reduce this to:

```suggestion
        tracing::debug!("`up` unimplemented");
```

Reply:
Updated